fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/executor/Rewriter.java

```diff
@@ -299,7 +299,8 @@ public class Rewriter extends AbstractBatchJobExecutor {
 
             topic("Eliminate GroupBy",
                     topDown(new EliminateGroupBy(),
-                            new MergeAggregate())
+                            new MergeAggregate(),
+                            new AdjustAggregateNullableForEmptySet())
```

Comment:
add comment here to explain why need execute AdjustAggregateNullableForEmptySet after MergeAggregate